2022-02-28 19:24:13,445 INFO     server.py Установлено соедение с клиентом ('127.0.0.1', 51482)
2022-02-28 19:24:13,451 DEBUG    decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51482)>,), {}. Вызов из модуля common.utils.Вызов из функции main().Вызов из функции main
2022-02-28 19:24:13,452 DEBUG    server.py Получено сообщение {'action': 'presence', 'time': 1646051053.4456835, 'user': {'account_name': 'Guest'}}
2022-02-28 19:24:13,452 DEBUG    server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646051053.4456835, 'user': {'account_name': 'Guest'}}
2022-02-28 19:24:13,452 DEBUG    decorator.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 1646051053.4456835, 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2022-02-28 19:24:13,452 INFO     server.py Сформирован ответ клиенту {'response': 200}
2022-02-28 19:24:13,452 DEBUG    decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51482)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции main().Вызов из функции main
2022-02-28 19:24:13,452 DEBUG    server.py Соединение с клиентом ('127.0.0.1', 51482) закрывается.
